api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -67,7 +68,7 @@
     /**
      * Configure the string value to a specified type
      * @param value the string representation of a property value
-     * @return the converted value
+     * @return the converted value or null if value is null
```

Comment:
+1 @OndrejM I defined it because most of the times null gets ignored and then people complain about things behaving undefined. Whether we finally need it or not - the behaviour is now defined!